src/index.ts

```diff
@@ -116,17 +120,24 @@ function resolveWebDependency(dep: string, isExplicit: boolean, isOptimized: boo
     );
   }
   if (dependencyStats.isFile()) {
-    return nodeModulesLoc;
+    const locations: DependencyLoc = {};
+    if (path.extname(dep) !== '.js') {
+      locations.styleLoc = nodeModulesLoc;
+    } else {
+      locations.depLoc = dep;
+    }
+    return locations;
   }
   if (dependencyStats.isDirectory()) {
     const dependencyManifestLoc = path.join(nodeModulesLoc, 'package.json');
     const manifest = require(dependencyManifestLoc);
     let foundEntrypoint: string = manifest.module;
+    let dependencyStyles: string = manifest.style;
```

Comment:
As discussed in #76, I'd prefer if this was backed out for the initial/experimental release. I'm sure we'll eventually end up going this direction, but I'd rather not support any sugar/shortcuts until I get a better sense of how people want to use this.